tests/images/Dockerfile.canary

```diff
@@ -24,7 +24,7 @@ RUN curl --silent --location "https://github.com/weaveworks/eksctl/releases/down
 # Install kustomize
 RUN curl -s https://api.github.com/repos/kubernetes-sigs/kustomize/releases |\
   grep browser_download |\
-  grep linux |\
+  grep linux_amd64 |\
   cut -d '"' -f 4 |\
   grep /kustomize/v |\
   sort | tail -n 1 |\
```

Comment:
We really should just be hardcoding this. I hate this so much lol

Reply:
right, synced with integ test container